setup.cfg

```diff
@@ -1,2 +1,42 @@
+[metadata]
+name = QtPy
+description = Provides an abstraction layer on top of the various Qt bindings (PyQt5, PyQt4 and PySide) and additional custom QWidgets.
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/spyder-ide/qtpy
+author = Colin Duquesnoy, The Spyder Development Team
+author_email = goanpeca@gmail.com
+maintainer = Gonzalo Peña-Castellanos
+maintainer_email = goanpeca@gmail.com
+license = MIT
+license_file = LICENSE.txt
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Environment :: Win32 (MS Windows)
+    Environment :: X11 Applications :: Qt
+    Intended Audience :: Developers
+    License :: OSI Approved :: MIT License
+    Operating System :: OS Independent
+    Programming Language :: Python :: 2
+    Programming Language :: Python :: 2.7
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.3
+    Programming Language :: Python :: 3.4
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
```

Comment:
these are the only supported python versions

Reply:
3.9 support is in https://github.com/spyder-ide/qtpy/pull/204

Reply:
Do we really still support Python 3.3 and 3.4? They're long-obsolete, Spyder itself dropped them ages ago.

Reply:
This should be resolved in PR #208 ; you'll need to re-run your tooling once that's merged since it changes a bunch of the metadata